lib/tasks/argo_testing.rake

```diff
@@ -62,7 +62,15 @@ if ['test', 'development'].include? Rails.env
 
         require 'webmock'
         # only allow connections to Fcrepo, Solr and workflow
-        WebMock.disable_net_connect!(allow: ['workflow:3000', 'solr:8983', 'fcrepo:8080', 'localhost:8983', 'localhost:8984', 'localhost:3004'])
+        WebMock.disable_net_connect!(allow: ['workflow:3000',
+                                             'solr:8983',
+                                             'fcrepo:8080',
+                                             'localhost:8983',
+                                             'localhost:8984',
+                                             'localhost:3004',
+                                             'dor-indexing-app:3000',
+                                             'dor-services-app:3000',
```

Comment:
I'm not sure why loading in fedora and reindexing requires argo to connect to dor-services-app.

Reply:
removed